SeQuant/core/asy_cost.hpp

```diff
@@ -13,14 +14,24 @@ class AsyCost {
   class AsyCostEntry {
```

Comment:
@bimalgaudel please add brief description (dox) to this and other classes ... it may make sense to merge your factorization branch into this branch first, then work on this branch

Reply:
what does `Asy` stand for btw?

Reply:
Asy stands for Asymptotic. I added dox.